src/transformers/models/marian/tokenization_marian.py

```diff
@@ -267,6 +269,9 @@ def decode(self, token_ids, **kwargs):
 
     def convert_tokens_to_string(self, tokens: List[str]) -> str:
         """Uses source spm if _decode_use_source_tokenizer is True, and target spm otherwise"""
+        if tokens[0].startswith(SPIECE_UNDERLINE):
+            tokens[0] = tokens[0][1:]
+
```

Comment:
Is this section necessary still?

Reply:
Yes, I accidentally left it there! Fixed it now.